packages/feedback-service/package.json

```diff
@@ -5,7 +5,7 @@
     "className": "Feedback",
     "serviceName": "feedback"
   },
-  "version": "0.0.9",
+  "version": "0.1.9",
```

Comment:
This should be `0.1.0`, not `0.1.9`.
Use `npm version minor` to bump versions. Do not manually change the versions.

Reply:
Change updated